README.md

```diff
@@ -8,23 +8,26 @@ This repository contains the source code and documentation for [thenewboston.com
 
 ## Contributing
 
-Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any 
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any
 contributions you make are **greatly appreciated**.
 
 1. Fork the project
 2. Create your feature branch
-3. Commit your changes
-4. Push to the branch
-5. Open a pull request
+3. run `npm run lint` and make sure you have no errors.
```

Comment:
This is the only instruction that starts with a lowercase and also has a period at the end